make chat window

kyohei-T opened the pull request:
# WHAT
チャット画面の作成

# WHY
サービスのメイン機能のため

Commit:
Apply haml

bundle install
 gem “haml”,”erb2haml”
.erb→.haml

Commit:
make chat window

edit message/index.html.haml

kyohei-T commented:
> # WHAT
> チャット画面の作成
> 
> # WHY
> サービスのメイン機能のため

チャット画面のSS
<img width="1439" alt="ChatSpace_と_Skitchの問題レポート" src="https://user-images.githubusercontent.com/48401497/54411708-65e63280-4733-11e9-9422-8c8ce16234c8.png">

hzak2xx reviewed (commented):
ハッシュロケットはシンボルに変更しましょう

Commit:
Edit  message/index.html.haml

ハッシュロケットで記載していた部分をシンボルによる記載に全て修正いたしました。

kyohei-T commented:
> ハッシュロケットはシンボルに変更しましょう

ご指摘ありがとうございます。全てのハッシュロケットをシンボルの記載に変更いたしました。
また、画面表示が変わらないことも確認いたしました。
チャット画面のSS(再取得）
<img width="1439" alt="ChatSpace_と_kyohei-T_chat-space" src="https://user-images.githubusercontent.com/48401497/54412661-f83c0580-4736-11e9-8077-c57409dd83b2.png">

TOMARUMARU commented on app/views/layouts/application.html.haml:
```diff
@@ -0,0 +1,10 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
```
ハッシュロケットこちらもですね

kyohei-T replied:
ご指摘ありがとうございます。修正いたしました。

TOMARUMARU commented on app/views/layouts/mailer.html.haml:
```diff
@@ -0,0 +1,8 @@
+!!!
```
mailerファイル使用しないので消しましょう

kyohei-T replied:
削除いたしました。

TOMARUMARU commented on app/views/layouts/mailer.text.haml:
```diff
@@ -0,0 +1 @@
+= yield
```
mailerファイル使用しないので消しましょう

kyohei-T replied:
削除いたしました。

TOMARUMARU commented on config/routes.rb:
```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
+  root 'messages#index'
+  get 'messages/index'
```
rootにあるのでこちらは不要です

kyohei-T replied:
get 'message/index'を削除いたしました。

Commit:
corrected some points

edit routes.rb, application.html.haml
remove mailer.html.haml, mailer.text.haml

TOMARUMARU reviewed (approved):
LGTM

kyohei-T commented:
> LGTM

ご確認ありがとうございます。
マージいたします。
